### ISSUE TITLE
configure script doesn't support Ubuntu 16.04

### ISSUE THREAD

**Issue by wcoppens** [redis/RedisDesktopManager#3660]
RDM Version: 0.8.7<br>
Environment (OS name and version): Ubuntu 16.04<br>
Redis Server Version: 3.2.1<br>

Steps to reproduce:
1. Execute ubuntu build commands from documentation (http://docs.redisdesktop.com/en/latest/install/#build-from-source)
2. `git clone --recursive https://github.com/uglide/RedisDesktopManager.git -b 0.8.0 rdm && cd ./rdm`
3. `cd src/`
4. `./configure`
   results in: 

```
E: Unable to locate package qt54base
E: Unable to locate package qt54imageformats
E: Unable to locate package qt54tools
E: Unable to locate package qt54declarative
E: Unable to locate package qt54quickcontrols
E: Unable to locate package qt54script
E: Unable to locate package qt54quick1
```

but tells me finally this:

```
====================================================================
Run: source /opt/qt54/bin/qt54-env.sh && qmake && make
====================================================================
```
1. Run from documentation: `source /opt/qt55/bin/qt55-env.sh && make && sudo make install` does not work result: `bash: /opt/qt55/bin/qt55-env.sh: No such file or directory`.

Same goes for `source /opt/qt54/bin/qt54-env.sh && qmake && make`

Expected result: 
Expected that packages qt54\* should be found. 
Expected `source /opt/qt54/bin/qt54-env.sh && qmake && make` should work.

Actual Result: 
Missing qt54\* packages
Not working `source` command.

**Comment by uglide** [comment]
Hello @wcoppens 
`configure` script doesn't support Ubuntu 16, you can update it by yourself or buy subscription and download ready-to-use deb package from [redisdesktop.com](https://redisdesktop.com/)

**Comment by wcoppens** [comment]
@uglide no problem, maybe update the documentation to prevent these kind of questions in the future ;-)

**Comment by jonathanpmartins** [comment]
Please, update your documentation! Lost of time...

**Comment by uglide** [comment]
@jonathanpmartins it's OSS, if you see that documentation should be updated - update it by yourself and submit PR.

Order subscription if you don't want to waste your time on building from source,  or spend 30 minutes or so to figure out what's wrong with configure script. From my perspective, it's a fair deal.

**Comment by uglide** [comment]
Fixed in 0.8.8
